addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderServerBridgeHandler.java

```diff
@@ -511,63 +511,50 @@ public synchronized Boolean isConnected() {
     }
 
     public boolean isOnline() {
-        return _online;
-    }
-
-    private synchronized boolean getConnected() {
-        return this.connected;
+        ThingStatusInfo statusInfo = getThing().getStatusInfo();
+        return (statusInfo.getStatus() == ThingStatus.ONLINE) ? true : false;
     }
 
     /**
      * Set connection status.
      *
      * @param connected
+     * @throws ZoneMinderUrlNotFoundException
```

Comment:
If you add this, please also mention a little bit about when the caller can expect them, and imho runtime exception do n not have to be mentioned there